code/game/gamemodes/objective.dm

```diff
@@ -154,7 +156,7 @@
 	if(!target || !considered_alive(target) || considered_afk(target))
 		return TRUE
 	var/turf/T = get_turf(target.current)
-	return T && !is_station_level(T.z)
+	return !T || !is_station_level(T.z)
```

Comment:
This is so that anyone who is sent Into The Sunset is considered exiled.

This objective currently counts people who escape to CentCom as exiled, so it should do the same for Into The Sunset.